package.json

```diff
@@ -69,7 +69,7 @@
     "react-test-renderer": "^15.5.4"
   },
   "scripts": {
-    "prepublish": "npm run lint && npm run test && npm run build",
+    "prepare": "npm run lint && npm run test && npm run build",
```

Comment:
Is prepublish deprecated? Why this change, just out of curiosity ?

Reply:
Ya sorry, `As of npm@5, `prepublish` scripts are deprecated.` was the warning. Here's more on that. https://docs.npmjs.com/misc/scripts#prepublish-and-prepare

Reply:
It's worth noting this was just a warning and could be done in a future PR when it's more of an error. `prepare` seems to have been around in npm a while so there should be a concern of that hook not being available.

Reply:
Ah that's right. This is good then, thanks! 👍